src/docs/010_manual/20_install.adoc

```diff
@@ -146,71 +246,74 @@ To get started with docToolchain, run `tasks` as the first command:
 ----
 .\dtcw.ps1 tasks --group=doctoolchain
 ----
+
 --
 
 .Windows with cmd.exe
 [role='secondary']
 --
+
 [source, cmd]
 ----
 dtcw.bat tasks --group=doctoolchain
 ----
 --
 
-This command shows all docToolchain tasks and verifies that docToolchain is installed by doing the following:
-
-* It checks for a configuration file and creates one if it does not exist. Prefers a local installation. 
-
-* It makes use of Docker (if available) where there is no local installation.
-
-NOTE: If this is your first time installing docToolchain, downloading might take a while.
-
-* If Docker is not available, it downloads and installs docToolchain to your home folder.
-** It then runs a Gradle build through the Gradle wrapper. If this is your first time, it will download and install Gradle.
-** The Gradle build also fetches some dependencies. This will also take a while if run for the first time.
+--
 
-NOTE: If you are behind a corporate proxy, you might need to consider build-script dependencies are fetched from a repository referenced by the property `mavenRepository`. By default the value `https://plugins.gradle.org/m2/` is used. When a repository requiring credentials is used the properties `mavenUsername` and `mavenPassword` can be set as well. 
-//in which configuration path/file? 
+NOTE: If you are behind a corporate proxy, you might need to consider build-script dependencies are fetched from a repository referenced by the property `mavenRepository`.
+By default the value `https://plugins.gradle.org/m2/` is used. When a repository requiring credentials is used the properties `mavenUsername` and `mavenPassword` can be set as well.
+//in which configuration path/file?
 
-[source, sh] 
+[source, sh]
 .Example command passing a custom maven repository with credentials from the command line
 ----
-./gradlew generateHTML -PmavenRepository=your_maven_repo -PmavenUsername=your_username -PmavenPassword=your_pw --info
+DTC_OPTS="-PmavenRepository=your_maven_repo -PmavenUsername=your_username -PmavenPassword=your_pw" ./dtcw tasks --group=doctoolchain --info
 ----
 
-You'll notice the installation is an incremental process and will take a while because of all the additional dependencies needed.
-After the first few commands, you'll have all the required dependencies installed and things will run faster.
-
 //end::install[]
 
-== Configure docToolChain to Use Existing Documents
+== Configure docToolchain to Use Existing Documents
 
```

Comment:
@rdmueller Lookig at the netlify preview sections after here are not rendered correctly. No clue why.

Reply:
found it. fixed it.
Seems that the `--` blocks where nested wrong